app/Repositories/Cts/BookingRepository.php

```diff
@@ -8,6 +8,11 @@
 
 class BookingRepository
 {
+    public function getBookings($perPage = 50)
```

Comment:
A general comment on this architectural decision: I'd like to see the pagination done in the HTTP layer so we can re-use this repository method in future to quite simply get a list of bookings.